### ISSUE TITLE
Bug: Cannot delete user

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#163]
```
Internal Server Error

The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```
![grafik](https://github.com/ail-project/ail-framework/assets/42176839/16cf9824-9935-4632-bae7-940330233898)

**Comment by Terrtia** [comment]
Hi @certrik  !
Thanks for the report !
``redis.exceptions.DataError: Invalid input of type: 'NoneType'. Convert to a bytes, string, int or float first`` Should be fixed with b38819f 

Can you check if you have another error in ``logs/ail.log``, ``logs/flask.log`` or ``logs/modules.log`` ?

**Comment by certrik** [comment]
It works now and I don't see any more errors. THX!
